src/components/views/dialogs/BaseDialog.tsx

```diff
@@ -174,7 +174,7 @@ export default class BaseDialog extends React.Component<IProps> {
                     >
                         {!!(this.props.title || headerImage) && (
                             <Heading
-                                size="h2"
+                                size="h4"
```

Comment:
Does this not violate https://dequeuniversity.com/rules/axe/4.4/heading-order

Reply:
Fixed too

Reply:
This mixing h2 and h4 is really confusing and a bad smell IMO